### ISSUE TITLE
Error when running bootnet with raw data

### ISSUE THREAD

**Issue by IreneBobevski** [SachaEpskamp/bootnet#51]
I am new to both R and network analysis. I am using EGA to estimate a network of psychological symptoms and I want to use bootnet to estimate the centrality stability with node dropping. However, I could only run bootnet after using estimateNetwork. I tried with the raw data, as it says in the documentation, but got an error message. What would be the reason for this? Am I doing the right thing?

This is my EGA estimation code:

gega.v1 <- EGA(gcombdata.v1, model = "glasso", algorithm = "louvain")

This code resulted in an error:

gega_centrstab_t4 <- bootnet(gcombdata.v1, nBoots=2500, type="case", nCores=3,
                                      statistics = c("strength", "closeness", "betweenness"))

"Note: bootnet will store only the following statistics:  strength, closeness, betweenness
Estimating sample network...
Error in do.call(.input$estimator, c(list(data), .input$arguments)) : 
  'what' must be a function or character string
In addition: Warning messages:
1: In formals(fun) : argument is not a function
2: In formals(fun) : argument is not a function"

So I run this instead, which worked:

gega_estnet_v1 <- estimateNetwork(gcombdata.v1, default="EBICglasso")
gega_centrstab_v1 <- bootnet(gega_estnet_v1, nBoots=2500, type="case", nCores=3,
          statistics = c("strength", "closeness", "betweenness"))
plot(gega_centrstab_v1, statistics = c("strength"), perNode=TRUE, labels=TRUE,            legend=TRUE)

Another issue is that after running bootnet the order of the strength centrality estimates in the node dropping plot was somewhat different compared to that estimated from the EGA network:     

gega_graph_v1 <- plot(gega.v1, vsize = 7, labels = colnames(gega_colnames_v1))
gega_centrstab_t4 <- bootnet(gcombdata_v1, nBoots=2500, type="case", nCores=3,
                                      statistics = c("strength", "closeness", "betweenness"))

Would this difference be because I run estimateNetwork or could this occur anyway because of the bootstrap sampling?

I am using R 4.0.2 with RStudio 1.3.959.

**Comment by SachaEpskamp** [comment]
Hi Irene,

Bootnet is supposed to be used only after running estimateNetwork, so that is correct. EGA has a few different arguments used under the good than bootnet (I think the lambda min ratio is different), hence the differences.

Best, Sacha
